support/k8s/k8s-workload-registrar/mode-crd/README.md

```diff
@@ -60,14 +59,16 @@ The configuration file is a **required** by the registrar. It contains
 This quick start sets up the SPIRE Server, SPIRE Agent, and CRD Kubernetes Workload Registrar.
 
 1. Deploy SPIRE Server, Kubernetes Workload Registrar, SPIRE Agent, and CRD. SPIRE Server and Kubernetes Workload Registrar will be deployed in the same Pod.
-   ```
+
+   ```shell
    kubectl apply -f https://raw.githubusercontent.com/spiffe/spire/main/support/k8s/k8s-workload-registrar/mode-crd/config/spiffeid.spiffe.io_spiffeids.yaml \
```

Comment:
For consistency, can we add `$` here? There are many other instances. We should either use `$` or (`#` where appropriate) or nothing. It's weird to see both in the same file.